src/pydata_sphinx_theme/theme/pydata_sphinx_theme/icon-links.html

```diff
@@ -1,22 +1,28 @@
-{%- macro icon_link_nav_item(url, icon, name) -%}
+{%- macro icon_link_nav_item(url, icon, name, type) -%}
   {%- if url | length > 2 %}
         <li class="nav-item">
           <a class="nav-link" href="{{ url }}" rel="noopener" target="_blank" title="{{ _(name) }}">
+            {%- if type == "fontawesome" -%}
             <span><i class="{{ icon }}"></i></span>
             <label class="sr-only">{{ _(name) }}</label>
+            {%- elif type == "local" -%}
+            <img src="{{ pathto(icon, 1) }}" class="icon-link-image" alt="{{ _(name) }}"/>
```

Comment:
`pathto` is sphinx sauce, right?

Reply:
yep: https://www.sphinx-doc.org/en/master/templating.html#pathto